scoring/templates/scoring/council.html

```diff
@@ -191,23 +189,23 @@ <h3 class="exclamation text-white">Visit us again on a bigger screen</h3>
                         </tbody>
                     </table>
                     <div class="table-footer mt-2">
-                        <a class="mt-3" href="">See results in context</a>
+                        <a class="mt-3" href="">See results in full list</a>
```

Comment:
@lucascumsille where should this link be pointing to? The "homepage" for this council type?

(If so, `{% url 'scoring' council.get_scoring_group.slug %}` will do it.)